### ISSUE TITLE
Shapes inside a group lose attributes, if multipass is on

### ISSUE THREAD

**Issue by bonfish** [imagemin/imagemin-svgo#10]
I'm using imagemin-svgo v 4.1.2 with this set of plugins:
```
	{cleanupIDs:{minify:false}},
	{removeRasterImages:false},
	{removeTitle:false},
	{removeXMLProcInst:false},
	{cleanupNumericValues:{floatPrecision:0}},
```
and `multipass: true` .The original SVG (**№1**):
```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!-- Generator: Adobe Illustrator 15.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg xmlns="http://www.w3.org/2000/svg" height="113.27" width="64.468" version="1.1" y="0px" x="0px" viewBox="0 0 64.468004 113.27">
	 <g>
		<polygon fill="#ff0000" points="421.95,36.272,399.62,41.947,389.78,124.81,411.44,124.81" transform="translate(-381.124,-11.541001)"/>
 		 <polygon fill="#ff0000" points="381.12,28.182,445.55,11.811,445.59,11.541,383.46,11.541" transform="translate(-381.124,-11.541001)"/>
	</g>
</svg>
```
is processed into (**№2**):
```
<?xml version="1.0" encoding="UTF-8" standalone="no"?><svg xmlns="http://www.w3.org/2000/svg" height="113" width="64" version="1" viewBox="0 0 64.468004 113.27"><path d="M40.826 24.73l-22.33 5.676-9.84 82.863h21.66zm-40.83-8.09L64.426.27l.04-.27H2.336z"/></svg>
```
The problem is that `fill` attribute for both polygons is lost. I thought it was an issue with SVGO, but then tried turning `multipass` off. The result was this (note the fill bubbling up to a group) (**№2.1**):
```
<?xml version="1.0" encoding="UTF-8" standalone="no"?><svg xmlns="http://www.w3.org/2000/svg" height="113" width="64" version="1" viewBox="0 0 64.468004 113.27"><g fill="#f00"><path d="M40.826 24.73l-22.33 5.676-9.84 82.863h21.66zM-.004 16.64L64.426.27l.04-.27H2.336z"/></g></svg>
```
And if it is processed with imagemin-svgo again, the result is the same compressed shape, but with the fill kept (**№2.2**)
```
<?xml version="1.0" encoding="UTF-8" standalone="no"?><svg xmlns="http://www.w3.org/2000/svg" height="113" width="64" version="1" viewBox="0 0 64.468004 113.27"><path d="M40.826 24.73l-22.33 5.676-9.84 82.863h21.66zm-40.83-8.09L64.426.27l.04-.27H2.336z" fill="#f00"/></svg>
```
--------------
in short: imgmin-svgo with `multipass:true` processes **№1** into **№2** instead of **№2.2**.

But the most interesting part is that if we run the module with multipass: true, even **№2.1** is turned into **№2**, i.e. **the group with a fill is totally lost**.

**Comment by kevva** [comment]
You should open this at https://github.com/svg/svgo.

**Comment by steven313** [comment]
so i have been making banners for adwords and kept getting this error where it doesn't support sfw files so kept looking and kept looking for this file its referring to then it hit me that the svg also has some code and i found the sfw file within their (not to familiar with svg files) my question is would the svg still work if i just take out the code sfw ill past the code in down below if not what can i do the banners need to be scalable

<?xml version="1.0" encoding="utf-8"?>

<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" [
<!ENTITY ns_extend "http://ns.adobe.com/Extensibility/1.0/">
<!ENTITY ns_ai "http://ns.adobe.com/AdobeIllustrator/10.0/">
<!ENTITY ns_graphs "http://ns.adobe.com/Graphs/1.0/">
<!ENTITY ns_vars "http://ns.adobe.com/Variables/1.0/">
<!ENTITY ns_imrep "http://ns.adobe.com/ImageReplacement/1.0/">
<!ENTITY ns_sfw "http://ns.adobe.com/SaveForWeb/1.0/">
<!ENTITY ns_custom "http://ns.adobe.com/GenericCustomNamespace/1.0/">
<!ENTITY ns_adobe_xpath "http://ns.adobe.com/XPath/1.0/">
]>
xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="300px" height="50px"

<![CDATA[ .st0{font-family:'Rockwell-Bold';} .st1{font-size:17.0079;} .st2{fill:#231F20;} .st3{font-family:'Rockwell';} .st4{letter-spacing:59;} .st5{font-size:8.5039;} ]]>






AT THE CONVERSE STORE
GATEWAY SHOP G275
please help ASAP 
Regards steven
